.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+name: test
+on: [push]
+
+env:
+  LIBZIM_RELEASE: libzim_linux-x86_64-6.1.1
+  LIBZIM_LIBRARY_PATH: lib/x86_64-linux-gnu/libzim.so.6.1.1
+  LIBZIM_INCLUDE_PATH: include/zim
+  CYTHON_VERSION: 0.29.6
+  MAX_LINE_LENGTH: 110
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+          architecture: x64
+
+      - name: Autoformat with black
+        run: |
+          pip install black
+          black . --exclude=setup.py
```

Comment:
This should fails if black reformat files.
If you just reformat the files on the CI it is useless.

Reply:
yup, I know, see the `this can be made stricter later` comment below. we're going to upgrade it to fail on reformatting in a later commit, we just wanted to make sure the rest of the CI process can pass successfully, we don't intend to keep it this way

Reply:
these will now fail on format errors